src/MfaVerifyController.js

```diff
@@ -100,6 +115,14 @@ function (Okta, Checkbox, BaseLoginController, TOTPForm, YubikeyForm, SecurityQu
           this.options.appState.get('isMfaRequired')) {
         return true;
       }
+      if (this.options.appState.get('isSuccessResponse')) {
```

Comment:
This is still not going to work - check out this diagram:
http://developer.okta.com/docs/api/resources/authn.html#transaction-state

After MFA_CHALLENGE, we can't assume that we're going to hit SUCCESS (even if the user has accepted the challenge). We can hit PASSWORD_WARN or PASSWORD_EXPIRED, which I'd imagine should still save their cookie value. I think the other times I've brought this up I've added some suggestions to work around this - ask me if you have any questions.